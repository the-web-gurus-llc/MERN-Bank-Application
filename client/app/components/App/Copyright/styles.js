import styled from 'styled-components';

export const StyledCopyright = styled.div`
  text-align: right;
  padding: 12px 24px;
  font-size: 13px;

  a {
    text-decoration: underline;
  }
`;
