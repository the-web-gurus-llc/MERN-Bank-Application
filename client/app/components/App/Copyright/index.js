import React from 'react';
import { StyledCopyright } from './styles';

export default function Copyright() {
  return (
    <StyledCopyright>
      Copyright &copy; {new Date().getFullYear()}. Made by{' '}
      <a href="https://github.com/pietrzakadrian" target="_blank">
        Adrian Pietrzak
      </a>
    </StyledCopyright>
  );
}
