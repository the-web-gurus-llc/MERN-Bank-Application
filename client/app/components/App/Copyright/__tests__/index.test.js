import React from 'react';
import { render } from '@testing-library/react';

import Copyright from '../index';

describe('<Copyright />', () => {
  it('should render a Copyright', () => {
    const { container } = render(<Copyright />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
