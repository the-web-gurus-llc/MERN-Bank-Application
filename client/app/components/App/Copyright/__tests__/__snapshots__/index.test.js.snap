// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Copyright /> should render a Copyright 1`] = `
<div
  class="Copyrightstyle__StyledCopyright-sc-1dw2rfe-0 kybcIo"
>
  Copyright Â© 
  2020
  . Made by
   
  <a
    href="https://github.com/pietrzakadrian"
    target="_blank"
  >
    Adrian Pietrzak
  </a>
</div>
`;
